### ISSUE TITLE
Bump and release?

### ISSUE THREAD

**Issue by nevinera** [shortdudey123/yamllint#50]
The published version of this gem still uses trollop - I see you have that updated already in the repository, and I'd love to drop the extra (rotting) dependency from our project :-)

Is there a release slated soon?

**Comment by jamiemccarthy** [comment]
Bumping this request 😁 

(My company's app has had this in our CI for some months now, pinned at 0f34496db4d6e42806eb5ee1d8fa2dfbcffe6ddd )

**Comment by shortdudey123** [comment]
Let me see if I can get this done.

**Comment by st0ck** [comment]
Yeah, that would be really awesome to have the new release with all the recent updates!

**Comment by marek130** [comment]
Hi guys, Any progress ? Many thanks!

**Comment by cvquesty** [comment]
Also needing the update to use optimist instead of trollop.  Please work on this if you can.
